test-projects/simple-database/project.visivo.yml

```diff
@@ -5,7 +5,7 @@ includes:
   - path: visivo-io/example-include.git@main
 
 defaults:
-  target_name: local-sqlite
+  target_name: local-postgres
```

Comment:
Hmm, looking at this I wonder if we should actually keep the `-t` option for run and serve even though we discussed removing it.  

It's actually pretty useful to be able to pass a target in the command line to use instead of the default in instances like these where you can't use env vars because sqlite and postgres have different connection requirements, but might occasionally want to do a run with a different target.

Reply:
Separately I think this change is failing the [deploy-dashboard](https://github.com/visivo-io/visivo/actions/runs/7647316700/workflow?pr=49) action. I'm not sure why the postgres test is failing tho.

Reply:
Good point, I was mostly talking about removing the ability to pass in a json target, which is already gone.  I kept the "default target".